xclim/core/calendar.py

```diff
@@ -379,7 +382,9 @@ def _convert_datetime(
         return np.nan
 
 
-def ensure_cftime_array(time: Sequence):
+def ensure_cftime_array(
+    time: Sequence,
+) -> Union[cftime.DatetimeGregorian, numpy.ndarray]:
```

Comment:
```suggestion
) -> Union[CFTimeIndex, numpy.ndarray]:
```
Not clear from the code itself, but I think those are the only two possibilities!